### ISSUE TITLE
gptune spack test fails on perlmutter

### ISSUE THREAD

**Issue by wspear** [E4S-Project/testsuite#48]
@liuyangzhuan

The spack test for this spec installed under e4s 22.11 fails on perlmutter.
```
-- linux-sles15-zen3 / gcc@11.2.0 -------------------------------
gptune@3.0.0~hypre~ipo+mpispawn~superlu build_system=cmake build_type=RelWithDebInfo
```

The test doesn't produce any console output.

**Comment by liuyangzhuan** [comment]
Is there any output file that is generated? Is this related to https://github.com/spack/spack/issues/32402?

**Comment by wspear** [comment]
@liuyangzhuan No output. You're right that might still be a result of it timing out. Does gptune need to be built with ~mpispawn on cray? We just realized that it's trying to use openmpi on our cray deployments which probably isn't right. That could be a/the problem.

**Comment by liuyangzhuan** [comment]
You are right. You should disable openmpi and mpispawn on Perlmutter. Can you try craympich?
